code/datums/diseases/_MobProcs.dm

```diff
@@ -39,7 +39,7 @@
 		var/datum/disease/DD = new D.type(1, D, 0)
 		viruses += DD
 		DD.affected_mob = src
-		DD.holder = src
+		SSdisease.active_diseases += DD //Add it to the active diseases list, now that it's actually in a mob and being processed.
```

Comment:
I didn't put this in `New` because it seemed redundant for a disease to be created, then it fails to contract, only for it to be remove from the list, again.